Reachability.m

```diff
@@ -98,8 +98,9 @@ +(instancetype)reachabilityWithHostname:(NSString*)hostname
     SCNetworkReachabilityRef ref = SCNetworkReachabilityCreateWithName(NULL, [hostname UTF8String]);
     if (ref) 
     {
-        id reachability = [[self alloc] initWithReachabilityRef:ref];
-
+        id reachability = [[self alloc] initWithReachabilityRef:CFBridgingRetain((__bridge id)ref)];
+	CFRelease(ref);
```

Comment:
Hey there, I saw your fix and wanted to leave a little feedback. I've been monitoring this issue closely I think you're the fourth person now :)

===

At line 98 retain count of the ref will be 1, due to the create.

We don't know immediately what the init does, so lets assume it doesn't retain it.

Now here on 102, you release ref. Its retain count is now 0. You will have created a zombie

(Final note) The init doesn't retain it, so all the above assumptions are true. Try and run the app with your changes, the reachability class will hit a zombie. See my solution https://github.com/tonymillion/Reachability/pull/118 for the full fix. You are correct in that you whenever you create a foundational item you should release it. However what you, and a couple others, ran into was the assumption that the init takes ownership. Which it doesn't at the moment. If you update the init to take ownership then your fix will be complete. :)